rmf_schedule_visualizer/src/VisualizerData.hpp

```diff
@@ -103,6 +108,9 @@ class VisualizerDataNode : public rclcpp::Node
   std::unordered_map<
     rmf_traffic::schedule::Version,
     std::vector<rmf_traffic::schedule::ParticipantId>> _conflicts;
+
+  rmf_traffic_ros2::schedule::WriterPtr _writer;
```

Comment:
This writer appears to be completely unused. That makes sense since this is only a visualizer; it should never need to write to the schedule database. Since it isn't (and shouldn't) be used, I suggest we delete it.

Reply:
removed  3b1c580